docs/faq.rst

```diff
@@ -169,6 +178,10 @@ Quick example: ::
     if emoji:
         await message.add_reaction(emoji)
 
+    # if you have the name and ID of a custom emoji:
+    emoji = 'python3:232720527448342530'
```

Comment:
Realistically speaking this entire example being added mainly works by accident and is a quirk in Discord's behaviour. In the future the library will probably just remove <...> from the string in the future. There could be something about using the much more self-documenting `discord.PartialEmoji` class but that's still in the interim at the moment.